pkgs/applications/science/math/qalculate-qt/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, intltool, pkg-config, qmake, wrapQtAppsHook, libqalculate, qtbase, qttools }:
+
+stdenv.mkDerivation rec {
+  pname = "qalculate-qt";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "qalculate";
+    repo = "qalculate-qt";
+    rev = "v${version}";
+    sha256 = "sha256-7H1nQLJBiuTj/GwojfOPpRbDseOHvLa94LK+bXvLhws=";
+  };
+
+  nativeBuildInputs = [ qmake intltool pkg-config wrapQtAppsHook ];
+  buildInputs = [ libqalculate qtbase qttools ];
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    description = "The ultimate desktop calculator";
+    homepage = "http://qalculate.github.io";
+    maintainers = with maintainers; [ _4825764518 ];
+    license = licenses.gpl2Plus;
+    platforms = [ "x86_64-linux" ];
```

Comment:
Maybe the platform can be more general, like `unix` or `linux`.

Reply:
Doesn't seem like it. It fails to build on aarch64-linux for me and ofborg. So let's go back to `x86_64-linux` and get this merged.